### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by est77**

### CODE REVIEW

File: src/appleseed/renderer/modeling/object/meshobjectoperations.cpp [hunk 1]
```diff
@@ -258,13 +258,13 @@ void compute_signature(MurmurHash& hash, const MeshObject& object)
     for (size_t j = 0, je = object.get_motion_segment_count(); j < je; ++j)
     {
         for (size_t i = 0, e = object.get_vertex_count(); i < e; ++i)
-            object.get_vertex_pose(i, j);
+            hash.append(object.get_vertex_pose(i, j));
```
Comment (dictoon) [inline src/appleseed/renderer/modeling/object/meshobjectoperations.cpp]: Completely missed that during my review...

File: src/appleseed/foundation/utility/murmurhash.cpp [hunk 2]
```diff
@@ -92,7 +92,7 @@ void MurmurHash::append(const void* data, const size_t bytes)
 
     // tail
 
-    const uint64 * tail = (reinterpret_cast<const uint64*>(data)) + nBlocks*16;
+    const uint8* tail = reinterpret_cast<const uint8*>(data) + nBlocks*16;
```
Comment (dictoon) [inline src/appleseed/foundation/utility/murmurhash.cpp]: Can we add unit tests for MurmurHash that compare our hashes against known ones?

File: src/appleseed/renderer/modeling/object/meshobjectoperations.cpp (hunk 1 above)
Comment (est77) [reply src/appleseed/renderer/modeling/object/meshobjectoperations.cpp]: And in the Maya plugin too, where the code comes from.

File: src/appleseed/foundation/utility/murmurhash.cpp (hunk 2 above)
Comment (est77) [reply src/appleseed/foundation/utility/murmurhash.cpp]: Done

File: src/appleseed/foundation/meta/tests/test_murmurhash.cpp [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+TEST_SUITE(Foundation_Utility_MurmurHash)
+{
+    TEST_CASE(Append)
+    {
+        MurmurHash hash;
+        EXPECT_EQ(hash.to_string(), "00000000000000000000000000000000");
+
+        hash.append("La llama que llama");
+        EXPECT_EQ(hash.to_string(), "e28f5add49177087aec4a6584d3f145c");
```
Comment (dictoon) [inline src/appleseed/foundation/meta/tests/test_murmurhash.cpp]: Expected values must come first (important for failure messages).

**Pull request merged**